R/describe-feature.R

```diff
@@ -53,7 +54,16 @@ bcdc_describe_feature.default <- function(record) {
 bcdc_describe_feature.character <- function(record){
 
   if (is_whse_object_name(record)) {
-    return(feature_helper(record))
+    bgc <- bcdc_get_capabilities()
+    cat_record <- bcdc_get_record(bgc$cat_url[grepl(record, bgc$whse_name)])
+
+    return(
+      dplyr::left_join(
+        feature_helper(record),
+         cat_record$details[,c("column_comments", "column_name")],
+         by = c("col_name" = "column_name")
+       )
```

Comment:
Should this be abstracted out to a function since it's used here and in the `bcdc_record` method?